websites/A/Animea/presence.ts

```diff
@@ -0,0 +1,131 @@
+const presence = new Presence({ clientId: "1006563035957182545" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video = {
+	current: 0,
+	duration: 0,
+	paused: true,
+};
+
+presence.on(
+	"iFrameData",
+	(data: { current: number; duration: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp,
+		},
+		pathnameArray = document.location.pathname.split("/"),
+		[time, showCover, showButtons] = await Promise.all([
+			presence.getSetting<boolean>("time"),
+			presence.getSetting<boolean>("cover"),
+			presence.getSetting<boolean>("buttons"),
+		]);
+
+	switch (pathnameArray[1]) {
+		case "":
+			presenceData.details = "Parcours la page d'accueil";
+			break;
+		case "discovery": {
+			presenceData.details = "Découvre de nouvelles animations";
```

Comment:
```suggestion
			presenceData.details = "Découvre de nouveaux animes";
```

Reply:
Resolved in 401f95334db21f79fa5f8bbe9d999798d7fab508